package.json

```diff
@@ -10,43 +10,80 @@
   "author": "Salute Frontend Team <salute.developers@gmail.com>",
   "contributors": [],
   "type": "module",
-  "main": "lib/index.js",
+  "exports": {
```

Comment:
Is there other way to do this except list all files?

Reply:
I've managed to simplify this